books/free-programming-books.md

```diff
@@ -735,6 +735,7 @@
 * [freeCodeCamp](https://www.freecodecamp.org)
 * [FutureLearn](https://www.futurelearn.com)
 * [MIT OCW](http://ocw.mit.edu)
+* [MOOC.fi](https://www.mooc.fi/en/)
```

Comment:
Coud you remove this?

You submit it in:
- #5567

Reply:
Hey, I'm a beginner trying to learn about git and GitHub. How do I remove this ?

Reply:
1. Work on your forked branch: https://github.com/Nandan-18/free-programming-books/blob/my-branch3/books/free-programming-books.md
2. Find file / line and remove it.
3. Commit. Changes will be automatically reflected in PR. like you see what happens with the other suggestions.